bin/get_nam_status.pl

```diff
@@ -54,9 +54,56 @@
 # also set reasonable defaults
 ASGSUtil::setParameter( $jshash_ref, \$backsite,  "siteHost", "ftp.ncep.noaa.gov");
 ASGSUtil::setParameter( $jshash_ref, \$backdir,   "siteDir",  "/pub/data/nccf/com/nam/prod");
+# use values from JSON if they were provided
 $jshash_ref->{"siteHost"} = $backsite;
 $jshash_ref->{"siteDir"} = $backdir;
 #
+#  N O W C A S T   F R O M   F I L E S Y S T E M
+#
+if ( $backsite eq "filesystem" ) {
+   ASGSUtil::appMessage( "INFO", "Constructing nowcast from grib2 files found in $backdir.");
+   # check to see if the Operator-supplied directory exists
+   if ( -d $backdir ) {
+      ASGSUtil::appMessage( "INFO", "The directory $backdir was found.");
+   } else {
+      ASGSUtil::appMessage( "INFO", "The directory $backdir was not found.");
+      die;
+   }
```

Comment:
```suggestion
      my $msg = qq{"The directory $backdir was not found."};
      ASGSUtil::appMessage( "INFO", $msg );
      die $msg;
   }
```

I think `die` should have a message here since it's use is going to cause an error visible via STDERR anyway.